src/middlewared/middlewared/alert/source/freenas_bmc.py

```diff
@@ -10,7 +10,7 @@
 
 class FreeNASBMCAlertSource(ThreadedAlertSource):
     level = AlertLevel.CRITICAL
-    title = "FreeNAS Mini Critical IPMI Firmware Update Available"
+    title = "Critical Firmware Update Available"
```

Comment:
Is there anyway to hide this from the trueNAS customers? This will never be applicable to them.

Reply:
Done